prime-angular-frontend/src/app/modules/adjudication/shared/components/enrollee-table/enrollee-table.component.html

```diff
@@ -142,6 +149,11 @@
                       matTooltip="Enrollee flagged for manual adjudication"
                       matTooltipPosition="after">flag
             </mat-icon>
+            <mat-icon *ngIf="row.confirmed"
```

Comment:
![image](https://user-images.githubusercontent.com/59669288/118053207-f2eb9d00-b338-11eb-9417-af7368a2b4d1.png)
Just a knit picky thing but this icon seems to shift the text down and make it miss-aligned. Im assuming the icon above causes the same thing but might be something worth addressing.

Reply:
Hahaha beat you to the nitpicky :)